core/src/main/java/com/sequenceiq/cloudbreak/job/stackpatcher/ExistingStackPatcherConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.cloudbreak.job.stackpatcher;
+
+import javax.annotation.PostConstruct;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ExistingStackPatcherConfig {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExistingStackPatcherConfig.class);
+
+    @Value("${existingstackpatcher.intervalhours}")
```

Comment:
This should have a default value.

Reply:
As discussed offline, the service will not start without providing a value so it can stay like that